bin/casperjs

```diff
@@ -124,7 +124,7 @@ for arg in SYS_ARGS:
     if not found and arg_name != 'engine':
         CASPER_ARGS.append(arg)
 
-CASPER_COMMAND = ENGINE_EXECUTABLE.split(' ')
+CASPER_COMMAND = [ENGINE_EXECUTABLE]
```

Comment:
I'm not a Windows guy, but it should be the same for the win exe (src/casperjs.cs, line 145), and maybe solve issue #269?